app/operations/application_operation.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationOperation
+  def self.call(*args)
+    new(*args).call
```

Comment:
@ailecksandr I agree, operations are ok, but we already have interactors and services at the beginning was planned as small-operations. I am trying to keep architecture clean and not allow to generate layers of indirections.